### PR TITLE
A set of cleanup

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
speed.c needs a bit more digging and work to be fully fixed... 
`cert_chain` seems to be a relic...

### CODE REVIEW

**Comment by richsalz** [comment]
+1

**Comment by FdaSilvaYY** [comment]
@richsalz : I swap  a short commit from another PR into this one, after your +1 .
It is very similar to other changes here.

**Comment by richsalz** [comment]
Reconfirm +1

File: apps/speed.c [hunk 1]
```diff
@@ -1033,11 +1033,12 @@ static int ECDSA_verify_loop(void *args)
     }
     return count;
 }
-
+/* TBC: outlen is globally shared, in a strange way*/
```
Comment (mattcaswell) [inline apps/speed.c]: Ping @agrandi, I think this is to do with the your async changes. Should this be a member of `loopargs_t`?

Comment (FdaSilvaYY) [inline apps/speed.c]: Fixes are on my patch queue, I will PR it soon.
The next variable also should be pushed into `loopargs_t`

Comment (FdaSilvaYY) [inline apps/speed.c]: I take this change out and pack it with more similar ones into #1342

**Comment by mattcaswell** [comment]
+1

File: apps/speed.c (hunk 1 above)
Comment (agrandi) [inline apps/speed.c]: @mattcaswell I'll add my comments directly to #1342

**Comment by richsalz** [comment]
merged ,thank you.

**Pull request closed without merging**